lstchain/reco/dl0_to_dl1.py

```diff
@@ -260,31 +308,37 @@ def r0_to_dl1(input_filename=get_dataset_path('gamma_test_large.simtel.gz'), out
 
                     dl1_container.prefix = tel.prefix
 
-
                     extra_im.tel_id = telescope_id
-                    for container in [extra_im, dl1_container, event.dl0, tel]:
+                    for container in [extra_im, dl1_container, event.r0, tel]:
                         add_global_metadata(container, metadata)
 
+                    event.r0.prefix = ''
+
                     writer.write(table_name=f'telescope/image/{tel_name}',
-                                 containers=[event.dl0, tel, extra_im])
+                                 containers=[event.r0, tel, extra_im])
```

Comment:
Shouldn't we write `event.dl1` container with images and pulse times here instead of `event.r0` or `event.dl0` as you had before? This is something I doubted about when I was writing UCM script

Reply:
The images and pulse time come from `tel`. `event.r0` includes info such as `obs_id` and `event_id`

Reply:
That's true